test/controllers/home.js

```diff
@@ -0,0 +1,29 @@
+const supertest = require('supertest');
+const tape = require('tape');
+const app = require('../../src/app');
+
+
+tape('testing the home route content type and status code', (t) => {
+  supertest(app)
+    .get('/home')
+    .expect(200)
+    .expect('Content-Type', 'text/html; charset=utf-8')
+    .end((err, res) => {
+      t.error(err);
+      t.equal(Object.values(res.headers).indexOf('text/html; charset=utf-8') > -1, true, 'content type should be html');
+      t.end();
+    });
+});
+
+
+tape('testing the home route technology', (t) => {
+  supertest(app)
+    .get('/home')
+    .expect(200)
+    .expect('x-powered-by', 'Express')
```

Comment:
This is basically testing for anything, because you don't have an API yet, you're testing for things like the contents of some page or it's headers, who cares about that? If the server is running as it should be - which it isn't - you should have some real stuff to test for.